packages/ui-extensions/src/surfaces/point-of-sale/types/cart.ts

```diff
@@ -107,3 +108,84 @@ export interface Address {
   provinceCode?: string;
   countryCode?: CountryCode;
 }
+
+export type ShippingLine =
+  | {
+      handle: string;
+      price: string;
+      title: string;
+      methodType: ShippingMethodType;
+      type: 'Calculated';
+    }
+  | {
+      handle: string;
+      price: string;
+      title: string;
+      type: 'Custom';
+    };
+
+export enum ShippingMethodType {
+  Shipping = 'SHIPPING',
+  Retail = 'RETAIL',
+}
+
+export interface TaxLine {
```

Comment:
POS model of [TaxLine](https://github.com/Shopify/pos-next-react-native/blob/50520de03bd020dcc7fb90bf3cc99b089f9817ad/src/native_libraries/CheckoutSDK/models/TaxLine.ts#L3-L12)